### ISSUE TITLE
Emotion and Event queries are always constant - SenseVoiceSmall

### ISSUE THREAD

**Issue by BNarayanaReddy** [modelscope/FunASR#2962]
Notice: In order to resolve issues efficiently, please follow the template and include reproducible details.
（注意：为了更加高效率解决您遇到的问题，请按照模板提问，补充可复现细节。）

## 🐛 Bug

The emotion and the event query should be taken from the input datasets, but in the sensevoice encoder class at model.py is taking a constant values everytime, leading to no use of emotion and event condition to the model.

[SenseVoiceSmall/model.py](https://github.com/modelscope/FunASR/blob/516c4f770496a5cbb89c8e2e447211bbb7b0db71/funasr/models/sense_voice/model.py#L860)

## To Reproduce

Steps to reproduce the behavior. Always include the exact command you ran.

1. Install with: `...`
2. Run: `...`
3. See error: `...`

## Code sample

```python
event_emo_query = self.embed(torch.LongTensor([[1, 2]]).to(speech.device)).repeat(
            speech.size(0), 1, 1
        )
input_query = torch.cat((language_query, event_emo_query), dim=1)
speech = torch.cat((input_query, speech), dim=1)
speech_lengths += 3

```

## Error logs

```text

```

## Environment

- OS:
- Python version:
- FunASR version:
- ModelScope version:
- PyTorch / torchaudio version:
- Install method (`pip`, source, Docker):
- Device (`cuda`, `cpu`, `mps`):
- GPU model:
- CUDA/cuDNN version:
- Docker image tag, if used:

## Audio details

If the audio cannot be shared, please describe:

- Duration:
- Sample rate:
- Format:
- Language/dialect:
- Speaker count:
- Background noise/music:

**Comment by LauraGPT** [comment]
Thanks for the detailed observation!

This is actually **by design**, not a bug. Let me explain the SenseVoice architecture:

**How SenseVoice works:**

The input sequence to the encoder is structured as:
```
[language_query] [event_query] [emo_query] [style_query] [speech_features...]
```

- `language_query` — Set dynamically based on detected/specified language
- `event_query` (token ID=1) and `emo_query` (token ID=2) — These are **placeholder tokens** that serve as "query slots" for the model to attend to
- `style_query` — Set based on the `textnorm` parameter

**The key insight:** Emotion and event are **outputs**, not inputs. The model predicts emotion/event labels as part of the CTC output sequence, conditioned on the speech features. The fixed query tokens `[1, 2]` act as positional anchors that the model has learned to associate with emotion/event prediction during training.

Think of it like a `[CLS]` token in BERT — it's always the same token, but the model learns to encode task-relevant information at that position.

**If you want to condition on specific emotion/event (e.g., for conditional generation):** This model architecture doesn't support that — it's designed for multi-task *recognition* (predicting emotion/event from speech), not conditional generation.

Hope this clarifies!
